src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -109,7 +110,35 @@ def run(self, revision):
             '-p', 'obj-x86_64-pc-linux-gnu/',
             '-checks={}'.format(','.join(checks)),
         ] + modified_files
-        clang_output = run_check(cmd, cwd=self.repo_dir)
-
-        # TODO Analyse clang output
-        logger.info('Clang output', output=clang_output)
+        clang_output = run_check(cmd, cwd=self.repo_dir).decode('utf-8')
```

Comment:
i wonder why we are not decoding this already in ``run_check``. might be worthwhile already doing this there.